app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -315,6 +315,7 @@ def pod_definition
               "--scan-type=openscap",
               "--serve=0.0.0.0:#{options[:pod_port]}"
             ],
+            :imagePullPolicy => "Always",
```

Comment:
@moolitayer just out of curiosity, what's the impact of moving `:imagePullPolicy` after `:image`?  Is it just for aesthetics?